statler_site/templates/play-detail-page.html

```diff
@@ -2,6 +2,18 @@
 
 {% block page-script %}
     <script>
+    
+    // on page load function
+    $( function()
+   {
+      // Test if a cookie exists
+      if(Cookies.get("post") != undefined)
```

Comment:
Nitpick: I think the `!= undefined` is unnecessary, because `undefined` is inherently [falsy](http://www.sitepoint.com/javascript-truthy-falsy/).

(You're welcome to ignore this and merge :-))